test/api_integration/apis/saved_objects/bulk_get.js

```diff
@@ -0,0 +1,122 @@
+import expect from 'expect.js';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const esArchiver = getService('esArchiver');
+
+  const BULK_REQUESTS = [
+    {
+      type: 'visualization',
+      id: 'dd7caf20-9efd-11e7-acb3-3dab96693fab',
+    },
+    {
+      type: 'dashboard',
+      id: 'does not exist',
+    },
+    {
+      type: 'config',
+      id: '7.0.0-alpha1',
+    },
+  ];
+
+  describe('bulk_get', () => {
+    describe('with kibana index', () => {
+      before(() => esArchiver.load('saved_objects/basic'));
+      after(() => esArchiver.unload('saved_objects/basic'));
+
+      it('should return 200 with individual responses', async () => (
+        await supertest
+          .post(`/api/saved_objects/bulk_get`)
+          .send(BULK_REQUESTS)
+          .expect(200)
+          .then(resp => {
+            expect(resp.body).to.eql({
+              saved_objects: [
```

Comment:
Would be :tada: to have snapshot tests for these, but that's of course not possible yet

Reply:
These are basically snapshot tests, no? What would be different with a snapshot test?

Reply:
++ Yeah, it's basically the same (I just prefer working with snapshots over large objects, e.g. when changing code etc)

Reply:
(so it wasn't really input on this PR, I was just spreading my love for snapshots 🎉 🙈)

Reply:
Haha, cool. Yeah, if we didn't have random data like auto generated ids it would be easier to do something like that

Reply:
Shit, I totally meant to delete my comment, thought it failed, and accidentally deleted yours 😿